src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketDelete.java

```diff
@@ -119,13 +128,80 @@ public void run(ActionContext context) throws Exception {
       if (fs.exists(gcsPath)) {
         try {
           fs.delete(gcsPath, true);
+          deleteCount++;
         } catch (IOException e) {
           LOG.warn(
             String.format("Failed to delete path '%s'", gcsPath)
           );
         }
       }
     }
+
+    for (GCSPath gcsPath : gcsPathsWild) {
+      String regex = getRegex(gcsPath);
+      try {
```

Comment:
does the logic of getting filesystem need to be in the loop ?

Reply:
If we have delete objects from several bucket, we should get filesystem use different path. I am gonna to clarify if we should support delete from different buckets